bin/rad-create-keys

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env radicle
+
+(load! (find-module-file! "prelude.rad"))
+(load! (find-module-file! "prelude/cmd-parsing.rad"))
```

Comment:
Can you import cmd-parsing from prelude, and then remove this line?

Reply:
I realised that I am actually not using cmd parsing here, so I removed it.